var/spack/repos/builtin/packages/totalview/package.py

```diff
@@ -19,67 +19,192 @@ class Totalview(Package):
     be used as the primary source and the architecture appropriate tarball will
     be downloaded as a resource."""
 
-    homepage = "https://www.roguewave.com/products-services/totalview"
-    maintainers = ["dshrader"]
+    homepage = "https://totalview.io"
+    maintainers = ["nicholas-sly", "dshrader", "petertea"]
```

Comment:
@nicholas-sly Are you okay with being added as a maintainer on this package?

Reply:
Since @nicholas-sly was removed from the maintainer list a while ago and he was ok with it then, and he hasn't responded here, let's go ahead and leave him off. @petertea, would you please go ahead and remove him from your branch?

Reply:
@dshrader I have just removed @nicholas-sly from the maintainer list